k8s/upgrades/jiva-0.6.0-0.8.1/jiva_volume_upgrade.sh

```diff
@@ -113,10 +113,17 @@ if [[ -z $c_name ]]; then
 fi
 
 controller_version=`kubectl get deployment $c_dep -n $ns -o jsonpath='{.metadata.labels.openebs\.io/version}'`
-
+if [[ "$controller_version" != "" ]] && [[ "$controller_version" != "$target_upgrade_version" ]]; then
+    echo "Current Target deployment $c_dep version is not $current_version or $target_upgrade_version";exit 1;
+fi
 replica_version=`kubectl get deployment $r_dep -n $ns -o jsonpath='{.metadata.labels.openebs\.io/version}'`
-
+if [[ "$replica_version" != "" ]] && [[ "$replica_version" != "$target_upgrade_version" ]]; then
```

Comment:
same as above and in below condition also

Reply:
fixed